src/models/fields/IllPerson.ts

```diff
@@ -0,0 +1,7 @@
+export enum IllPerson {
+    director = 'director',
+    accountant = 'accountant',
+    family = 'family',
+    employee = 'employee',
+    someoneElse = 'someoneElse'
```

Comment:
You are using otherPerson in a number of other places in your code, I think its worth going over those locations and trying to refactor.